src/components/button/Button.scss

```diff
@@ -0,0 +1,23 @@
+.button {
+  justify-self: center;
+  height: 40px;
+  border-width: 0px;
+  padding-inline: 30px;
+  color: orange;
+  font-weight: bold;
+
+  &:hover {
+    color: white;
+  }
+
+  &--primary {
+    border-radius: 25px;
+    background-color: white;
+    background-repeat: repeat;
+
+    &:hover {
+      border: 2px solid white;
+      background-color: orange;
+    }
+  }
+}
```

Comment:
Go to your VS code open the settings, look for `insert final newline` and enable it. 

It's a good practice to have the last line of the file empty, Github warn you about it, take a lok:
![Screen Shot 2022-04-06 at 8 52 01 AM](https://user-images.githubusercontent.com/2808008/161968679-151c8d75-483b-43ec-a713-1ba15a8806b1.png)
: